### DISCUSSION TITLE
Latest firmware EM7565 issues

### DISCUSSION THREAD

**Discussion by szyper-sakrona** [danielewood/sierra-wireless-modems#27] [General]
Hello,

I'd like to ask you what do you think about latest firmware of our modems (exactly em7565).
Yesterday I upgraded from previous to latest one:

-------(from ati output)------------
Model: EM7565
Revision: SWI9X50C_01.09.04.00 6d3218 jenkins 2019/02/26 20:33:22
-------------------------------------_

I was very interested if SierraWireless fixed CA issues that people reported...
But the effect of latest firmware (upgrade from windows EXE file, easier this time for me) is very odd...

In my country I used B3&B7 (B20 adds nothing to my dl/ul speed usually) CA.
Now I choose this band, check from at!band? command and guess what is in at!gstatus command:

_-----------------------------
at!gstatus?
!GSTATUS: 
Current Time:  34020		Temperature: 44
Reset Counter: 1		Mode:        ONLINE         
System mode:   LTE        	PS state:    Attached     
LTE band:      B7     		LTE bw:      15 MHz  
LTE Rx chan:   3175		LTE Tx chan: 21175
LTE SSC1 state:ACTIVE      	LTE SSC1 band: B2     
LTE SSC1 bw  : 10 MHz  		LTE SSC1 chan: 1599
LTE SSC2 state:ACTIVE      	LTE SSC2 band: B19    
LTE SSC2 bw  : 10 MHz  		LTE SSC2 chan: 6350
LTE SSC3 state:NOT ASSIGNED
LTE SSC4 state:NOT ASSIGNED
EMM state:     Registered     	Normal Service 
RRC state:     RRC Connected  
IMS reg state: No Srv  		

---------(cut)-------------------_
B7 (ok), B2????, B19?????    - on previous firmware the bands was displayed correctly now it's different...
But overall the transfer speeds are high (up to 100 Mb/s - that typical max speeds around night at my location) - but whats happenned  to bands. It should be B3, B7, B20...
What do you think I should check?
Regards
Bart

**Comment by ChilliWinter** [comment]
Same here; primary band is correct number, when the secondary show band -1 numbered
